boards/arm/s32z270dc2_r52/s32z270dc2_r52.dtsi

```diff
@@ -68,3 +68,23 @@
 &stm3 {
 	clock-frequency = <133333333>;
 };
+
+&swt0 {
+	clock-frequency = <48000000>;
```

Comment:
Should one of the watchdog timers be `status=okay`, so that it can be configured by applications? It seems the watchdog timers are currently only being set to `status=okay` in tests that utilize them